blocks/library/paragraph/index.js

```diff
@@ -248,6 +249,21 @@ registerBlockType( 'core/paragraph', {
 				),
 			},
 		],
+		to: [ 'left', 'center', 'right' ].map( ( value ) => ( {
+			type: 'shortcut',
+			shortcut: value.charAt( 0 ),
```

Comment:
Might be worth adding an inline comment; this is giving me clever code vibes.

Aside: Could this be `value[ 0 ]`?

Reply:
Maybe I'll do `map( { left: 'l' } )` etc. instead then... :) What do you mean with `value[ 0 ]`? It's not an array?

Reply:
Per MDN ([string, §Character access](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/string)), array-like access was introduced in ES5, so I assume it safe to use. Both forms should be equivalent.

Reply:
I had no idea this was possible. :)